kitchen.yml

```diff
@@ -4,11 +4,20 @@ verifier:
 
 provisioner:
   name: chef_zero
-  enforce_idempotency: true
-  multiple_converge: 2
-  deprecations_as_errors: true
+  data_bags_path: test/integration/data_bags
+  encrypted_data_bag_secret_key_path: test/integration/encrypted_data_bag_secret
```

Comment:
Let's stop using the encrypted data bags in test kitchen so that it's easier to deal with while testing. Can you please remove this and convert all of the encrypted databags to plain text json files?